plugins/woocommerce-blocks/packages/components/totals/item/style.scss

```diff
@@ -11,7 +11,7 @@
 
 .wc-block-components-totals-item__value {
 	font-weight: bold;
-	white-space: nowrap;
+	white-space: wrap;
```

Comment:
Hi @niharikamahajan02, first of all, thank you so much for contributing 💪🏼 

As for the proposed change, `wrap`  is not a valid value for the `white-space` property, I believe what you're looking for here is changing this to `normal` which will merge whitespaces and make the text wrap as necessary.
Furthermore, if you are getting the proper behaviour with the invalid value, it means that the inherited property is already set to `normal`, and if so you can achieve the desired behaviour by deleting line 14.

You read more about the `white-space` property [here](https://www.w3schools.com/cssref/pr_text_white-space.php)

Also, be sure to fill in the `Changelog entry ` details on any PR you submit, as this is a requirement for every change. I will fill this in for you as an example on this PR. Happy coding 😃